api/src/libs/logWriter.ts

```diff
@@ -39,25 +41,21 @@ export const LogWriter = (groupName: string, logStream: string, region?: string)
 
   const LogError = async (message: string, details?: AxiosError): Promise<void> => {
     try {
-      console.error(`${message} - ${JSON.stringify(details?.toJSON())}`);
+      isNotTestEnv ?? console.error(`${message} - ${JSON.stringify(details?.toJSON())}`);
```

Comment:
[pebble] Did you mean to use `||` here? `isNotTestEnv` will always be a boolean, so it'll never be undefined and therefore never trigger the other half of the nullish coalescing operator.